data/json/furniture_and_terrain/terrain-manufactured.json

```diff
@@ -1437,5 +1437,78 @@
     "move_cost": 3,
     "flags": [ "TRANSPARENT", "DIGGABLE", "MOUNTABLE", "NOCOLLIDE", "CONTAINER", "SEALED" ],
     "bash": { "sound": "thump", "ter_set": "t_dirt", "str_min": 50, "str_max": 100, "str_min_supported": 100 }
+  },
+  {
+    "type": "terrain",
+    "id": "t_street_light",
+    "name": "street light",
+    "looks_like": "t_utility_pole",
+    "description": "Raised source of light which used to illuminate streets and their surrounding area, but it's useless without electricity.",
+    "symbol": "T",
+    "color": "grey",
+    "move_cost": 0,
+    "coverage": 40,
+    "flags": [ "BASHABLE" ],
+    "bash": {
+      "str_min": 30,
+      "str_max": 100,
+      "sound": "metal screeching!",
+      "sound_fail": "clang!",
+      "ter_set": "t_thconc_floor",
+      "items": [
+        { "item": "steel_lump", "count": [ 0, 2 ] },
+        { "item": "steel_chunk", "count": [ 1, 4 ] },
+        { "item": "scrap", "count": [ 3, 12 ] },
+        { "item": "plastic_chunk", "count": [ 2, 4 ] }
```

Comment:
Could probably drop some suitable electronic components as well if broken, I think. Same for traffic light and utility pole.